libraries/stdlib/common/src/generated/_Maps.kt

```diff
@@ -58,6 +58,8 @@ public inline fun <K, V, R, C : MutableCollection<in R>> Map<out K, V>.flatMapTo
 /**
  * Returns a list containing the results of applying the given [transform] function
  * to each entry in the original map.
+ * 
+ * @sample samples.collections.Maps.Transformations.mapMap
```

Comment:
to many `map`s in the sample qualified name :)
`mapToList` would be better.

Reply:
Done